web/js/modules/vector-styles/reducers.js

```diff
@@ -14,9 +14,9 @@ export const defaultVectorStyleState = {
 };
 export function getInitialVectorStyleState(config) {
   const custom = lodashGet(config, 'vectorStyles') || {};
-  return lodashAssign({}, defaultVectorStyleState, {
-    custom,
-  });
+  const customDefault = lodashCloneDeep(custom);
```

Comment:
Just an FYI: [structuredClone](https://developer.mozilla.org/en-US/docs/Web/API/structuredClone) is now available natively and should do the same thing that `cloneDeep` from lodash does.